Dockerfile

```diff
@@ -1,29 +1,52 @@
 ARG node_version=20.13-alpine3.19
 
-FROM node:${node_version} as builder
+FROM --platform=$BUILDPLATFORM node:${node_version} as build
 RUN mkdir /opt/epp-client
 WORKDIR /opt/epp-client
-COPY .yarnrc.yml .yarnrc.yml
-COPY .env.yarn .env.yarn
-COPY .yarn/releases .yarn/releases
-COPY package.json package.json
-COPY yarn.lock yarn.lock
+COPY ./ ./
 RUN yarn
 
-FROM node:${node_version} as base
+FROM --platform=$BUILDPLATFORM build as production_build
+RUN yarn build
```

Comment:
Again, run at native on the build platform, we get this faster than when we rely on emulated platform build.